lib/src/layer/tile_layer/tile_provider/network_image_provider.dart

```diff
@@ -11,8 +11,7 @@ import 'package:http/http.dart';
 /// Note that specifying a [fallbackUrl] will prevent this image provider from
 /// being cached.
 @immutable
-class FlutterMapNetworkImageProvider
-    extends ImageProvider<FlutterMapNetworkImageProvider> {
+class MapNetworkImageProvider extends ImageProvider<MapNetworkImageProvider> {
```

Comment:
Additionally as @TesteurManiak said, some typedef deprecations are at least needed. `(Flutter)MapNetworkImageProvider` may be commonly used externally, for example.

Reply:
For real? `(Flutter)MapNetworkImageProvider` is not exported by the package therefore shouldn't be public in my opinion.
Are you referring to the unchanged `NetworkTileProvider` that is part of the public API?

Reply:
I can add deprecation typedefs for internal classes, but in what szenario are they used externally? They wouldn't have any usage internal too because the code is migrated to the new class names.

Reply:
Ah, sorry, thought I'd already responded here. No, I got it wrong, must have been half asleep when I reviewed this: of course FMNIP isn't used externally, that would be NTP. You're right, no reason to have typedefs for internal deprecations.